shared/constants/signup.tsx

```diff
@@ -29,7 +29,6 @@ type Store = {
   emailError: string
   emailVisible: boolean
   inviteCode: string
-  inviteCodeError: string
```

Comment:
the invite code flow is basically never visible and totally broken likely for a long time. instead of showing this we just show a generic error on any sad path